Source/WebKit/UIProcess/API/glib/WebKitUserContentManager.cpp

```diff
@@ -320,6 +454,36 @@ void webkit_user_content_manager_unregister_script_message_handler(WebKitUserCon
     manager->priv->userContentController->removeUserMessageHandlerForName(String::fromUTF8(name), API::ContentWorld::pageContentWorld());
 }
 
+/**
+ * webkit_user_content_manager_register_script_message_handler_with_reply:
+ * @manager: A #WebKitUserContentManager
+ * @name: Name of the script message channel
+ * @world_name: the name of a #WebKitScriptWorld
+ *
+ * Registers a new user script message handler in script world.
+ *
+ * Registers a new user script message handler in script world with name @world_name.
+ * If NULL is passed as the world_name, the default world will be used.
+ * See webkit_user_content_manager_register_script_message_handler() for full description.
```

Comment:
Reminder for a followup patch: when we deprecate the old functions, we may want to move the longer description here, and write `See webkit_user_content_manager_register_script_message_handler_with_reply() instead` in the deprecated ones.